specification/source/useCases.rst

```diff
@@ -616,6 +616,165 @@ Trigger                      Contractor notifies owner or PM that system is read
+---------------------------  ---------------------------------------------------
+Preconditions                1. Either a) whole building or system model for
+                             the particular design project, or b) sufficient information about the current state of the design, to enable the configuration of a model template
+                             based on a generic design for the appropriate building type. The model must be complete down to the required sensors and actuation points, which may be set-points for idealized local loop control, since the sequence only performs
+                             supervisory control.
+
+                             2. Control sequence to be assessed must match, or be capable of being configured to match, the
+                             building/system model in terms of sensors, set-points for idealized local loop control and
+                             modes of operation.
+
+                             3. Relevant statutory requirements and design performance targets, together with performance metrics derived from these requirements and
+                             targets.
+---------------------------  ---------------------------------------------------
+Successful End Condition     User is able to (i) compare the performance of
+                             different control sequences in terms of selected pre-defined criteria, and (ii) evaluate the ability of a selected control sequence to enable the building/system to meet or exceed externally-defined performance criteria.
+---------------------------  ---------------------------------------------------
+Failed End Condition         1. Building/system model or
+                             configuration information for generic model template is incomplete.
+
+                             2. Performance requirements or targets are incomplete or inconsistent wrt the specific
+                             control sequence.
+
+                             3. Simulation fails to run to completion or fails
+                             convergence tests.
+---------------------------  ---------------------------------------------------
+Primary Actors               Mechanical Designer/Consultant
+---------------------------  ---------------------------------------------------
+Secondary Actors
+---------------------------  ---------------------------------------------------
+Trigger                      Need to select or improve a control sequence for a
+                             building or system
+---------------------------  ---------------------------------------------------
+**Main Flow**                **Action**
+---------------------------  ---------------------------------------------------
+1                            User loads the building/system model for the
+                             project or uses design information to configure a model template: building type, system type(s), size …(?).
```

Comment:
Can you please clarify or remove "...(?)" as it otherwise does not contribute any information.